tests/test_search.py

```diff
@@ -198,9 +198,9 @@ def test_elasticsearch_exception(self):
             self.assertPostResponse(
                 path=url,
                 json_request_body=dict(es_query=smartseq2_paired_ends_query),
-                expected_code=requests.codes.internal_server_error,
-                expected_error=ExpectedErrorFields(code="internal_server_error",
-                                                   status=requests.codes.internal_server_error))
+                expected_code=requests.codes.not_found,
```

Comment:
Shouldn't this be a client error (maybe 400?)?

Reply:
The error is actually cause on the server side when the DSS_ES_ENDPOINT is removed. Maybe the error should be "Elasticsearch Misconfigured"? Either way, not_found seems like the appropriate error in my opinion.

Reply:
If it's a server error, look for a 5xx error code.

Reply:
How about 503 service_unavailable?